packages/components/addon/components/hds/tabs/index.hbs

```diff
@@ -1,36 +1,31 @@
-{{! template-lint-disable no-invalid-role }}
-{{! template-lint-disable require-context-role }}
-
-<div class={{this.classNames}} ...attributes>
-  {{! Hds-TabList }}
-  <div class="hds-tab-list hds-typography-body-200">
-    <ul class="hds-tab-list__list" role="tablist">
-      {{yield (hash Tab=(component "hds/tabs/tab"))}}
+<div class={{this.classNames}} {{did-insert this.didInsert}} ...attributes>
```

Comment:
[question] are you using `@ember/render-modifiers` here? Or is this a custom modifier?

Reply:
It was from an example @didoo had in a [test branch](https://github.com/hashicorp/design-system/tree/cristiano-test-tabs/packages/components/addon/components/hds/tabs). It's a modifier from `@ember/render-modifiers` that calls a function after its element is added to the DOM. Let me know if it would be better to use something else.